demo.jl

```diff
@@ -0,0 +1,67 @@
+using ImageFiltering, FFTW, LinearAlgebra, Profile, Random
```

Comment:
I saw the "temporarily" comment in the conversation, so I'm assuming this will change.